src/app/(Home)/(Domain)/domains/[domain]/[cluster]/[domainTab]/loading.tsx

```diff
@@ -0,0 +1,3 @@
+import SectionLoadingIndicator from '@/components/section-loading-indicator/section-loading-indicator';
+
+export default SectionLoadingIndicator;
```

Comment:
I'm out of context, but why do we need this?
As far as I can tell, it is just a proxy of the component imported from elsewhere. But I might be wrong since I'm unsure how the UI is built.